types/wlr_screencopy_v1.c

```diff
@@ -253,6 +269,23 @@ static void frame_handle_buffer_destroy(struct wl_listener *listener,
 	frame_destroy(frame);
 }
 
+static enum wl_shm_format fourcc_to_wl_shm_format(uint32_t fourcc) {
```

Comment:
Hmm. Can't we just compare formats without converting them? We shouldn't need these functions now.

Reply:
This one is needed to check if the supplied dmabuf format is correct. The other one is needed to convert from `frame->format` to fourcc.

Well, actually we can use the other one to check the format, but the way its done now is nicer, imo.